static/js/compose_state.js

```diff
@@ -37,13 +38,18 @@ function get_or_set(fieldname, keep_leading_whitespace) {
 }
 
 export function stream_name() {
-    return $("#stream_message_recipient_stream").val().trim();
+    let val = $("#stream_message_recipient_stream").text().trim();
+    if (val === "Select a stream") {
```

Comment:
It feels a bit messy to have the value be this in the UI and also be `""` in the layer below that, but I couldn't find a nicer way to do this (I forget why at this point I can look into it more).

Reply:
Ooo one possible solution is `compose_stream_widget.value()` -- I will try that.

Reply:
Huzzah! That works :)